packages/modsecurity/changelog.yml

```
@@ -1,4 +1,10 @@
 # newer versions go on top
+
+- version: "1.2.2"
```

```suggestion
- version: "1.3.0"
```
(enhancements will bump minor)